Rocket.Chat/Models/Attachment.swift

```diff
@@ -37,6 +37,7 @@ class Attachment: BaseModel {
         return titleLinkDownload && !titleLink.isEmpty
     }
 
+    @objc dynamic var identifier: String?
```

Comment:
If we have identifier here, why removing it as a `BaseModel`?

Reply:
We need it to do at least the in-memory cache until we close #1556, but it doesn't need to be indexed as a primary key this is why it's not a `BaseModel`

Reply:
Got it! 👍